ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf

```diff
@@ -5,7 +5,7 @@
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:06:00
 #PBS -l select=1:mpiprocs=48:ompthreads=1:ncpus=48:mem=48GB
-#PBS -l place=vscatter:shared
```

Comment:
Should the removal of `:shared` be mentioned in the release notes?

Reply:
Since they were never officially added to ops, no. They were included in the prior package for the obsproc.v1.2 upgrade for NCO's consideration....they said not now. Also, NCO didn't do a full install for that upgrade (they just did their usual copy of prior install and update obsproc_ver within) so the shared changes didn't make it into an ops install. This is us removing them from our side for now. Thanks for the question though!

Reply:
Sounds good, thanks for the explanation!